src/browser/generate-mock.ts

```diff
@@ -0,0 +1,29 @@
+import { type Mock } from "../helpers";
+import { type GenerateForBrowserResponse } from "../node/generateForBrowser";
+
+/**
+ * Generate a list of Mock objects from GenerateForBrowserResponse function
+ * @beta
+ */
+export async function generateMock(
+    list: GenerateForBrowserResponse[],
+): Promise<Mock[]> {
+    const mockList: Mock[] = [];
+
+    for (const item of list) {
+        let { default: mock } = (await import(item.filePath.toString())) as {
+            default: Mock | { default: Mock };
+        };
+
+        /* this should never really be required but a malformed "default"
+         * export in commonjs might (such as testcases in this repo does)
+         * would behave like this as it is not a proper default export */
+        if ("default" in mock) {
+            mock = mock.default;
+        }
+
+        mock.meta = { url: item.url, method: item.method };
+        mockList.push(mock);
+    }
+    return mockList;
```

Comment:
Föredra `Array.map()` över att loopa och pusha på nya element.

Dels för att det är en mikrooptimering (well jag är säker att V8 ändå optimerar detta men det blir enklare för javascript-motorn iaf eftersom den inte behöver gissa).

Men mest av allt för att det blir enklare för den mänskliga läsaren: med `Array.map()` ser jag direkt att avsikten med loopen är att för varje element transformera till ett nytt element. Med en for-slinga måste jag läsa hela koden för att hitta vad som händer på en högre nivå. Transformerar du? Filtrerar du? Reducerar du?

Nu kommer det ju isf bli en array med promises men det kan du hantera lätt med `await Promise.all()`

```ts
const mocks = list.map((item) => { ... });
return await Promise.all(mocks);
```